### PR TITLE
add com.remnantsoftheprecursors.ROTP

### PR DESCRIPTION
**Pull request by Aman9das**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.

The game assets are CC-BY-NC-ND-4.0, thus can be redistributed only for non commercial use.

- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://www.reddit.com/r/rotp/comments/sjl1q9/flatpak_packaging_success/
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
I was permitted by Ray Fowler to refer to the domain in app id.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

I created a PR but it's a very minor tweak, that can be added by hand too.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.remnantsoftheprecursors.ROTP.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.remnantsoftheprecursors.ROTP</id>
+  <name>Remnants of the Precursors</name>
+  <metadata_license> CC-BY-NC-ND-4.0</metadata_license>
```
Comment (hfiguiere) [inline com.remnantsoftheprecursors.ROTP.metainfo.xml:5]: this is not an accepted license for metadata. `CC0-1.0` is usually preferred.

Just to clarify this is not the game asset license. This is the license of THIS file.

File: com.remnantsoftheprecursors.ROTP.desktop [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Remnants of the Precursors
+Exec=Remnants.sh
+Type=Application
+Icon=com.remnantsoftheprecursors.ROTP
+Categories=Application;
```
Comment (hfiguiere) [inline com.remnantsoftheprecursors.ROTP.desktop:6]: Shouldn't there be a Game category?

**Comment by hfiguiere** [comment]
bot, build com.remnantsoftheprecursors.ROTP

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.remnantsoftheprecursors.ROTP.metainfo.xml (hunk 1 above)
Comment (Aman9das) [reply com.remnantsoftheprecursors.ROTP.metainfo.xml:5]: ok fixing asap

File: com.remnantsoftheprecursors.ROTP.desktop (hunk 2 above)
Comment (Aman9das) [reply com.remnantsoftheprecursors.ROTP.desktop]: Added in latest commit

File: com.remnantsoftheprecursors.ROTP.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    sources:
+    - type: file
+      path: Remnants.sh
+    - type: file
+      path: com.remnantsoftheprecursors.ROTP.png
+    - type: file
+      path: com.remnantsoftheprecursors.ROTP.desktop
+    - type: file
+      path: com.remnantsoftheprecursors.ROTP.metainfo.xml
+finish-args:
```
Comment (hfiguiere) [inline com.remnantsoftheprecursors.ROTP.yml:48]: finish-args are usually toward the top before the `modules`

File: com.remnantsoftheprecursors.ROTP.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  - --env=PATH=/app/jre/bin:/app/bin
+  - --env=JAVA_HOME=/app/jre
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline com.remnantsoftheprecursors.ROTP.yml:53]: does it support wayland?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.remnantsoftheprecursors.ROTP.yml (hunk 4 above)
Comment (Aman9das) [reply com.remnantsoftheprecursors.ROTP.yml:53]: It uses standard java libraries, so yes

Comment (hfiguiere) [reply com.remnantsoftheprecursors.ROTP.yml:53]: Then:

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

Comment (hfiguiere) [reply com.remnantsoftheprecursors.ROTP.yml:53]: unless there is a requirement to also have x11 (sometime it's the case)

Comment (Aman9das) [reply com.remnantsoftheprecursors.ROTP.yml]: I have made this specific change, let's hope nothing breaks..

File: com.remnantsoftheprecursors.ROTP.yml (hunk 3 above)
Comment (Aman9das) [reply com.remnantsoftheprecursors.ROTP.yml]: Sure! I was following the openjdk example.. my bad

**Comment by barthalion** [comment]
Hey @rayfowler, would you like to have write access to the Flathub repo? We do allow third-party submissions, but the goal always was to involve upstream developers.

**Comment by Aman9das** [comment]
bot, build com.remnantsoftheprecursors.ROTP

**Comment by rayfowler** [comment]
> Hey @rayfowler, would you like to have write access to the Flathub repo? We do allow third-party submissions, but the goal always was to involve upstream developers.

You guys can give me access if you'd like, but I have no way to test a Linux build

**Pull request closed without merging**